savant/meta/bbox.py

```diff
@@ -77,6 +78,20 @@ def tlbr(self) -> Tuple[float, float, float, float]:
         """[x_min, y_min, x_max, y_max] representation of the bbox."""
         return self.left, self.top, self.right, self.bottom
 
+    def polygon(self) -> np.ndarray:
+        """Representation of the bbox using 4 corner points.
+
+        :return: polygon np.array([[x1, y1], [x2, y2], [x3, y3], [x4, y4]]), value in float
+        """
+        return np.array(
+            [
+                [self.left, self.top],
+                [self.right, self.top],
+                [self.right, self.bottom],
+                [self.left, self.bottom],
+            ]
+        )
```

Comment:
```suggestion
        left, top, right, bottom = self.tlbr
        return np.array(
            [
                [left, top],
                [right, top],
                [right, bottom],
                [left, bottom],
            ]
        )
```

Cache left, top, right, bottom calculations